### PR TITLE
Feature backend

### PR DESCRIPTION
**Pull request by MashB**
Bug fix 
1. Regarding the table name change job_id
2. Scenario where multiple project id belongs to user was failing in GET request.

### CODE REVIEW

**Review by sureshgaussian — APPROVED** [review]
approved after code review

**Comment by MashB** [comment]
Branch went out of sync, synced now, need re-approval

**Review by sujata-m — APPROVED** [review]

**Pull request merged**
